cmd/server/openstack_iid_attestor/main_test.go

```diff
@@ -124,6 +131,142 @@ func TestAttest(t *testing.T) {
 	}
 }
 
+func TestMakeSelectorsFromSpiffeID(t *testing.T) {
```

Comment:
Since the plugin does NOT make the selectors from SPIFFE ID,
The following naming would be good.
```suggestion
func TestMakeSelectors(t *testing.T) {
```

Reply:
f9b81aa